monai/transforms/utility/array.py

```diff
@@ -281,33 +283,56 @@ def __call__(self, img: NdarrayOrTensor) -> NdarrayOrTensor:
         return img[:: self.repeats, :]
 
 
-class SplitChannel(Transform):
+class SplitDim(Transform):
     """
-    Split Numpy array or PyTorch Tensor data according to the channel dim.
-    It can help applying different following transforms to different channels.
+    Given an image of size X along a certain dimension, return a list of length X containing
+    images. Useful for converting 3D images into a stack of 2D images, for example.
 
-    Args:
-        channel_dim: which dimension of input image is the channel, default to 0.
+    Note: `torch.split`/`np.split` is used, so the outputs are views of the input (shallow copy).
 
+    Args:
+        dim: dimension on which to split
+        keepdim: if `True`, output will have singleton in the split dimension. If `False`, this
+            dimension will be squeezed.
     """
 
     backend = [TransformBackends.TORCH, TransformBackends.NUMPY]
 
-    def __init__(self, channel_dim: int = 0) -> None:
-        self.channel_dim = channel_dim
+    def __init__(self, dim: int = -1, keepdim: bool = True) -> None:
```

Comment:
May I know why change the default value from `0` to `-1`?

Thanks.

Reply:
No particularly strong reason, as the default could be anything. I went for `-1` as I thought the most common use-case would be converting 3d to 2d, so splitting along the depth dimension.

Reply:
OK, I see.
I feel `split channels` is also a very useful example, could you please help add some description for it in the "for example" of doc-string?

Thanks.